docs/ReleaseNotes.md

```diff
@@ -6,18 +6,52 @@ Acuminator 3.1 includes the bug fixes and enhancements described in this section
 
 ### Enhancements
 In Acuminator 3.1, the following enhancements have been implemented:
+ - The [PX1062](diagnostics/PX1062.md) diagnostic has been added to check that graphs and graph extensions do not contain statis members.
+ - The following new diagnostics have been added to prevent serealization and desealization errors: 
+   - The [PX1063](diagnostics/PX1063.md) diagnostic checks that the exception-derived classes always declare a serealization constructor.
+   - The [PX1064](diagnostics/PX1064.md) diagnostic checks that the exception-derived classes which declare new serealizable data, always provide an override of the `GetObjectData` method.
+ - Support for the following complex expressions passed to SetProcessDelegate API has been added to the [PX1008](diagnostics/PX1008.md) and [PX1086](diagnostics/PX1086.md) diagnostics:
```

Comment:
I would not call this change an enhancement, imho it's more close to a bugfix. But I'm not sure. @EAndrosova  what do you think?

Reply:
Why not?

Reply:
It really is a matter of taste. I think we can see many bug fixes as enhancements. So I will leave it up to you to decide where it should be placed. It doesn't look bad here.

In my opinion, the fact that these expressions were not supported originally is a bug. The analysis just was not correct in some cases because these cases were forgotten during the diagnostic creation. Moreover, it's not like we handled it gracefully, it was a run-time error in the analyzer.